cmd/server/main.go

```diff
@@ -26,6 +26,11 @@ func main() {
 			Value: "/var/lib/ocid/sandbox",
 			Usage: "ocid pod sandbox dir",
 		},
+		cli.StringFlag{
+			Name:  "runtime",
+			Value: "/usr/bin/runc",
```

Comment:
can we put just `runc` here and just use `os.LookPath` inside? I say this because Fedora ships under `/usr/bin` while the runc Makefile under `/usr/bin` or while developing I have runc under `$GOPATH/.../opencontainers/runc/runc` and I add that to my `$PATH` 

minor nit though